### DISCUSSION TITLE
iPhone WhatsApp not Indexed

### DISCUSSION THREAD

**Discussion by hpcnetworks** [sepinf-inc/IPED#2384] [Q&A]
Hi Guys!

I have already indexed iPhone WhatsApp contents a few times, after exporting iTunes Backup with iTunes Backup Explorer.

This time I backed up iPhone content with Magnet Axion and then exported it with iTunes Backup Explorer, as usual. 
Unfortunetely, after processing all content with IPED WhatsApp messages have not been processed.
I checked the WhatsApp folders and it seems to be okay, as seen on attached image.

Please, help me find out what is missing... I guess there is no difference on backuing up from iTunes or Magnet Axion, once both uses the same libraries to export all content.

Thanks in advance,

Bruno
![WhatsApp screenshot](https://github.com/user-attachments/assets/e20c6281-7021-45dc-a4c6-66a4bfa52d31)

![image](https://github.com/user-attachments/assets/0a289b97-38c4-4cb5-b6f5-5e091fd1eb33)
![WhatsApp screenshot](https://github.com/user-attachments/assets/eae9078e-2bc3-4870-a461-b73ba5a3492d)

**Comment by lfcnassif** [comment]
Please check if there is a ChatStorage.sqlite file in the exported backup and if it is not corrupted neither encrypted (try to open it with a SQLite reader app). If it's there and ok, please share the processing log.

**Comment by hpcnetworks** [comment]
Nassif, please forgive me!

All whatsapp chats are there. I processed it on a folder and opened another one. :(((

Sorry for wasting yout time!!!

Best Regards,

Bruno

**Reply by lfcnassif** [reply]
I'm sorry too, the selection on the pictures confused me.
